series/praha-pyvo/events/2019-09-18.yaml

```diff
@@ -9,24 +9,53 @@ venue: na-venecku
 urls: []
 talks:
 
-- title: Exceptions in Python
+- title: Výjimky
   speakers:
-  - ??
+  - Petr Viktorin
+  coverage:
+  - video: https://www.youtube.com/watch?v=gRaL4VwFph4
```

Comment:
I remember you've said: Don't record the slides, record me, I'm recording my screen.

Yet the video doesn't have your screen. What happened?

Reply:
The recording failed. In the beginning projector was showing weird colors, so we connected things up in a non-standard way, and the screen recording came out solid blue.
Thanks for not following my request; you saved the day!

Reply:
I actually tried to follow it. So I guess my poor recording skills were actually useful :+1: